frontend/src/pages/AboutPage.js

```diff
@@ -144,7 +158,16 @@ function AboutPage(props) {
         >
           /OMOPConceptSets?{codeset_ids.map(d => `codeset_ids=${d}`).join("&")}
         </Button>
-      <TextH2>How to make changes to a codeset (via Atlas JSON)</TextH2>
+      <TextH2>Database Refresh</TextH2>
```

Comment:
Looks good. IDK what 'contained' variant even is, I'll take a look later and see for myself.